xontrib/bashisms.py

```diff
@@ -105,6 +105,12 @@ def _set(args):
         __xonsh__.env["XONSH_TRACE_SUBPROC"] = True
     elif arg == "+x":
         __xonsh__.env["XONSH_TRACE_SUBPROC"] = False
+    elif arg in ["-ex", "-xe"]:
+        __xonsh__.env["RAISE_SUBPROC_ERROR"] = True
+        __xonsh__.env["XONSH_TRACE_SUBPROC"] = True
+    elif arg in ["+ex", "+xe"]:
+        __xonsh__.env["RAISE_SUBPROC_ERROR"] = False
+        __xonsh__.env["XONSH_TRACE_SUBPROC"] = False
```

Comment:
At some point it will make more sense to implement this with argparse, or a similar command line parsing interface

Reply:
I think that time is now :wink: